app/src/ereferrals/java/org/eyeseetea/malariacare/strategies/DashboardActivityStrategy.java

```diff
@@ -352,6 +352,11 @@ public void onSuccess(int syncedFiles) {
                     showToast(String.format("%d files synced", syncedFiles));
                 }
             }
+
+            @Override
+            public void onProgress() {
```

Comment:
this callback method should receive a parameter for indicate ProgressStatus as begin or end